### ISSUE TITLE
Allow for individual commands to be resent/canceled

### ISSUE THREAD

**Issue by freak-dk** [CraftingStore/Suggestions#772]
### Is your feature request related to a problem? Please describe.

Often I am finding myself wanting to either cancel a specific command related to a purchase and/or resend specific commands from a purchase and not all of them at once (which is possible atm. of course).

This is the case for me mostly when a customer has put in wrong details related to the purchase with the variables I have set up (What server to execute commands on etc.).

Solution right now, for me, is to delete the payment and recreate it which is a hassle and doesn't really seem intuitive having to put in all the details of the original payment again manually. 

### Describe the solution you'd like

Basically just an extra tab added on the commands page, where you're able to cancel/resend specific commands (Option would be either cancel/resend depending on whether the command has been executed or not yet).

https://gyazo.com/2e338b5cb8f62c09ce4cd7d4738f38bc

Looks a bit silly but should make sense :-]
